10up/insecure-content-warning
# Unlock post save when content changes

### Description of the Change

Currently we lock post saving when a post is saved and insecure items are found. Post saving is never unlocked unless the override checkbox is used. But if content changes are made directly to the post, we should unlock post saving to allow the checks to run again.

This PR addresses that by listening for content changes and if changes are made and we are currently in a locked state, we remove the lock.

### Alternate Designs

None

### Benefits

Saving is unlocked when changes are made

### Possible Drawbacks

None

### Verification Process

1. Create a post with an insecure item in it
2. Save the post
3. Notice saving is disabled and warnings are shown
4. Make some content changes to the post
5. Notice saving is now re-enabled

### Checklist:


### Changelog Entry

> Fix - ensure post saving is unlocked when content changes happen
